Source/WebCore/rendering/svg/SVGPathData.cpp

```diff
@@ -183,6 +184,25 @@ static Path pathFromRectElement(const SVGElement& element)
     return path;
 }
 
+static Path pathFromUseElement(const SVGElement& element)
+{
+    const auto& useElement = downcast<SVGUseElement>(element);
```

Comment:
We probably want to assert the downcast is safe.

Reply:
That's already done by the callers -- see the other methods, they work the same.